wfe2/wfe.go

```diff
@@ -2114,6 +2114,14 @@ func (wfe *WebFrontEndImpl) GetOrder(ctx context.Context, logEvent *web.RequestE
 	}
 
 	respObj := wfe.orderToOrderJSON(request, order)
+
+	// Our 99th percentile finalize latency is 2.3s. Asking clients to wait 3s
+	// before polling the order to get an updated status means that >99% of
+	// clients will fetch the updated order object exactly once,.
+	if respObj.Status == core.StatusProcessing {
+		response.Header().Set(headerRetryAfter, fmt.Sprintf("%d", 3))
```

Comment:
I would prefer that we stick to the same pattern we use for `wfe.sendError` + also using a constant. A constant would also ensure that' it's easier to test.

```suggestion
		response.Header().Set(headerRetryAfter, strconv.Itoa(fooRetryAfterSeconds))
```